<?xml version="1.0" ?>

<launch>

  <!-- Localization-->
  <node pkg="amcl" type="amcl" name="robot2_amcl" output="screen">
    <remap from="scan" to="robot2/scan"/>
    <rosparam file="$(find hosp_bot)/config/amcl2.yaml" command="load" />
    <remap from="initialpose"               to="/robot2/initialpose"/>
    <remap from="amcl_pose"                 to="/robot2/amcl_pose"/>
    <remap from="particlecloud"             to="/robot2/particlecloud"/>
  </node>

</launch>
